windows/src/desktop/kmshell/main/UfrmProxyConfiguration.dfm

```diff
@@ -2,11 +2,11 @@ inherited frmProxyConfiguration: TfrmProxyConfiguration
   BorderIcons = [biSystemMenu]
   BorderStyle = bsDialog
   Caption = 'frmProxyConfiguration'
-  ClientHeight = 288
-  ClientWidth = 428
```

Comment:
Will these be wide enough for localized strings?

Reply:
Hopefully -- it's not a big issue as it is very rarely used, and we could probably eliminate it altogether!

Reply:
I take it there are... reasons... we can't just detect the width of the label elements and dynamically adjust appropriately?  Or do the label definitions themselves have a fixed width, rather than fitting their contained text?

I can easily accept a fixed height, but I do share @darcywong00's concern about localized strings.

Reply:
Yeah, there's a bunch of reasons why we can't dynamically resize the dialog at present. Mostly because we have no code to do the dialog resize. Earlier versions of Keyman allowed for customisation of the dialog size and even customisation of the template itself for localization, but that functionality was never used and added a significant amount of complexity to the localization process. So for now, this is what we've got. Given this is not a new issue with this dialog, and this dialog is *very* rarely used, and this problem is non-trivial to resolve (it's not just here, really), I'd like to just not worry about it for now!